.vib/zookeeper/goss/goss.yaml

```
@@ -0,0 +1,28 @@
+file:
+  /opt/bitnami/zookeeper/logs:
+    exists: true
+    filetype: directory
+    mode: "0775"
+  /opt/bitnami/zookeeper/conf/zoo.cfg:
+    exists: true
+    filetype: file
+    mode: "0644"
+    contains:
+      - {{ printf "/clientPort=.*%s/" .Env.ZOO_PORT_NUMBER }}
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: []
+  zookeeper-version-ok:
+    exec: "/opt/bitnami/zookeeper/bin/zkCli.sh -server 127.0.0.1:$ZOO_PORT_NUMBER version | grep \"ZooKeeper CLI version: $APP_VERSION\""
```

Is there any reason to enclose this with quotes?

```suggestion
    exec: /opt/bitnami/zookeeper/bin/zkCli.sh -server 127.0.0.1:$ZOO_PORT_NUMBER version | grep "ZooKeeper CLI version: $APP_VERSION"
```

---

I got the "Nested mappings are not allowed in compact mappings" linter error if I don't use the quotes.

---

Then it is okay to use them haha, thanks for checking !